client/controllers/LobbyController.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import { LOBBY_MODE } from "../../configs/game";
+import { Host } from "../view/lobby";
+import Normal from "../view/lobby/Normal";
+
+interface Props {
+  type: LOBBY_MODE;
+}
+
+function Lobby({ type }: Props): JSX.Element {
+  function render(type: LOBBY_MODE): JSX.Element {
+    switch (type) {
```

Comment:
It's fine, but I think switch for one case is overkill, also render function may be unnecessary in this situation

Reply:
sure thing michael 🤝